### DISCUSSION TITLE
Self-Host Instance

### DISCUSSION THREAD

**Discussion by yuyux9** [stoatchat/discussions#532] [Q&A] [answered]
How to change file size limit that i can upload? By default it seems 20mb in autumn. Then i change it to 90 nothing happens. Does anyone know how to increase file size limit?

**Comment by AruMoon** [comment, marked as answer]
you also need to change it in revite client, it's hardcoded

**Reply by yuyux9** [reply]
thanks a lot! it works!

**Comment by steel4me** [comment]
@AruMoon @yuyux9 can you tell me where? i cant find it, maybe im blind

**Reply by AruMoon** [reply]
Here https://github.com/revoltchat/revite/blob/master/src/components/common/messaging/MessageBox.tsx#L578
And here https://github.com/revoltchat/revite/blob/master/src/components/common/messaging/MessageBox.tsx#L635

**Comment by steel4me** [comment]
thank you very much 👍

**Comment by CamjamPNG** [comment]
How would i edit it inside a docker server?
